.github/workflows/publish-npm-package.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '20'
+          node-version: '16'
```

Comment:
This version should be updated as well, to use a node.js 20 env in our pipelines
Any reason to leave it as 16?

Reply:
Sorry, seems like this workflow is not used even, I've just realized :D